dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -27,16 +27,21 @@ public class ObjectCreationNode extends Node {
     // Class body for anonymous classes, otherwise null.
     protected final @Nullable ClassDeclarationNode classbody;
 
+    // Enclosing type receiver
+    protected final Node receiver;
+
     public ObjectCreationNode(
```

Comment:
Also, to allow existing code to work, deprecate the original constructor and have it call the new constructor with a null argument. So existing clients don't break immediately.
Also, add a changelog entry.

Reply:
Thanks! Since there is only one usage of initializing ObjectCreationNode, we can keep current code.